### ISSUE TITLE
`repr(VARIANT.missing)` raises `NotImplementedError`

### ISSUE THREAD

**Issue by junkmd** [enthought/comtypes#372]
It is very inconvenient when debugging, so let's add `elif self is type(self).missing` to `__repr__`.

https://github.com/enthought/comtypes/blob/dbe4fa1bed94fef20b9c56faad1693230dd32c43/comtypes/automation.py#L205-L208

```py
    def __repr__(self):
        if self.vt & VT_BYREF:
            return "VARIANT(vt=0x%x, byref(%r))" % (self.vt, self[0])
        elif self is type(self).missing:
            return "VARIANT.missing"
        return "VARIANT(vt=0x%x, %r)" % (self.vt, self.value)
```

**Comment by junkmd** [comment]
I will work this while adding type hints to `comtypes.automation`.

**Comment by junkmd** [comment]
This was resolved by #374
